src/Controller/GraphQL/LoginController.php

```diff
@@ -57,7 +57,7 @@ public function __construct(UserProviderInterface $userProvider, UserPasswordHas
     /**
      * @Mutation()
```

Comment:
One more `#[Mutation]` should be here. :)

Reply:
Updated. Thanks.